### ISSUE TITLE
Check source elements audio/video tags for media

### ISSUE THREAD

**Issue by Xiuywft** [marklieberman/downloadstar#29]
Download Star doesn't find, and therefore won't download, mp3 files from https://exoticaproject.com/4/, whereas a right-click and "Save audio as..." in Firefox downloads them just fine, albeit one at a time.

The code looks like this, but I removed all the angle brackets, because five minutes of searching and reading didn't find me a way to quote a chunk of HTML without this window interpreting it

div class='record'
a name='agogos_tictactoe' href='images/agogos_tictactoe.jpg' title='The A Go-Gos, Tic-Tac-Toe'
img src='images_thumbnails/agogos_tictactoe.gif'
/a
div
audio id='player2' controls style='max-width:100%;'
source src='/4/audio/agogos_tictactoe.mp3' type='audio/mp3'
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
THIS line is not understood by Download Star

/audio
/div
The A Go-Gos: "Tic-Tac-Toe" br/
[DB-3665A] br/
br/
Motifs: 
a href='index.php?name=randb_cover' target='_self'
R&B cover
/a
a href='index.php?name=organ' target='_self' 
Organ
/a
a href='index.php?name=overt_booker_t_and_the_mgs_influence' target='_self'
Overt Booker T & MGs influence
/a
div style='position:absolute;left:1px;bottom:0px;font-size:8px;color:gray'
a href='/4/#agogos_tictactoe'
#
/a
/div
div style='position:absolute;right:1px;bottom:0px;font-size:8px;color:gray'
added: 2016-03-11
/div
...

**Comment by marklieberman** [comment]
Scrape.js currently doesn't look at source elements inside audio/video tags. That is an oversight and will have to be fixed.

**Comment by OkanEsen** [comment]
I just looked inside `scrape.js` and the scraping should work according to https://github.com/marklieberman/downloadstar/blob/master/src/content/scrape.js#L104.

I also just tested it on the same page with [these settings](https://user-images.githubusercontent.com/552226/41622571-4a31ec42-7410-11e8-92e7-a5ef3917b8c4.png) and the addon is finding every `.mp3` file on the page and downloads it successfully. 

Are you sure you're selecting the right option for the scraping @Xiuywft?

**Comment by ssokolow** [comment]
@Xiuywft 

To quote source code in Markdown, either indent it by four spaces or use "fenced code blocks". The latter look like this:

    ```html
    <html>
        <body>Blah blah blah.</body>
    </html>
    ```

...and the result looks like this:

```html
<html>
    <body>Blah blah blah.</body>
</html>
```

(To show what a fenced code block's raw source looks like, I also indented it by four spaces.)

**Comment by marklieberman** [comment]
Closed by 327f8faaede36b5a89cc4aae7a41fa735b06570d
